tool/dactyl-config.yml

```diff
@@ -203,6 +203,14 @@ pages:
             - local
             - ripple.com
 
+# Listing XRP as an Exchange
+    -   category: Tutorials
+        html: xrp-listing-guide.html
+        md: exchange-guide.md
```

Comment:
Please (1) make the HTML and MD filenames match as closely as possible to each other and to the title of the file and (2) follow the existing filename convention for consistency's sake.

Suggested change:

```yaml
html: tutorial-listing-xrp.html
md: tutorial-listing-xrp.md
```